include/tunnler/room_member.h

```diff
@@ -132,6 +132,12 @@ class RoomMember final {
     MacAddress mac_address; ///< The mac_address of this member.
 
     /**
+     * Extracts a chat entry from a received RakNet packet and adds it to the chat queue.
+     * @param packet The RakNet packet that was received.
+     */
+    void HandleChatPackets(const RakNet::Packet* packet);
```

Comment:
I've implemented this in my branch too (as I needed it) and decided to rename this and the WiFi func to `...Packet(` instead of `...Packets(` as it really only ever proceeds one packet at a time.